src/io.jl

```diff
@@ -95,23 +102,33 @@ function Base.write(io::IO, d::MRCData; compress=:none, unit_vsize=4096)
     data = parent(d)
     fswap = bswapfromh(h.machst)
     unit_vsize = div(unit_vsize, sizeof(T))
+    if isnothing(buffer)
```

Comment:
On older Julia versions at least, this is faster:
```suggestion
    if buffer === nothing
```